src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -507,6 +560,17 @@ private void verifyPresentInDatastore(FeedbackQuestionAttributes expectedQuestio
         assertEquals(gson.toJson(expectedQuestion), gson.toJson(actualQuestion));
     }
 
+    private void verifyPresentInDatastore(FeedbackResponseAttributes expectedResponse) {
+        String responseJsonString = BackDoor.getFeedbackResponseAsJson(expectedResponse.feedbackQuestionId,
+                                                                       expectedResponse.giverEmail,
+                                                                       expectedResponse.recipientEmail);
+        FeedbackResponseAttributes actualResponse = gson.fromJson(responseJsonString, FeedbackResponseAttributes.class);
+
+        // Match the id of the expected Feedback Response because it is not known in advance
```

Comment:
@unyoungwax, This isn't true, the IDs for feedback responses are known in advance